metrics/prometheus-agent/templates/prometheus.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: monitoring.coreos.com/v1
 kind: Prometheus
 metadata:
-  name: {{ template "prometheus-agent.name" . }} 
+  name: {{ template "prometheus-agent.name" . }}
   namespace: {{ template "prometheus-agent.namespace" . }}
   labels:
     app: {{ template "prometheus-agent.name" . }}
     app.kubernetes.io/name: {{ template "prometheus-agent.name" . }}
     app.kubernetes.io/component: prometheus
     release: {{ $.Release.Name | quote }}
 spec:
-  alerting: null
-  ruleNamespaceSelector: null
-  ruleSelector: null
-  walCompression: null
-  disableCompaction: null
-  retention: null
+  alerting: {}
+  ruleNamespaceSelector: {}
+  ruleSelector: {}
+  walCompression: true
+  disableCompaction: false
+  retention: 10d
```

Comment:
Why 10days? This is agent. It should not keep data at all (only the first 2 hours).

Reply:
no probs it was the default values, I will change it